scripts/monitor.sh

```diff
@@ -1,96 +1,99 @@
 #!/bin/bash
 
-# Specify the base directory
+# Define the base directory for the NFC tracking application.
 BASE_DIR="/home/potato/NFC_Tracking"
 
-# Specify the name and path of your C programs
+# Define the names and paths of the C programs to be monitored.
 PROGRAM_NAME_1="read_ultralight"
```

Comment:
Consider using arrays to manage the program names and paths. This approach will make the script more concise, reduce potential errors from copy-pasting, and make it easier to add or remove programs in the future. 

```diff
- PROGRAM_NAME_1="read_ultralight"
- PROGRAM_NAME_2="button_listener"
- PROGRAM_NAME_3="screen.py"
- PROGRAM_PATH_1="$BASE_DIR/$PROGRAM_NAME_1"
- PROGRAM_PATH_2="$BASE_DIR/$PROGRAM_NAME_2"
- PROGRAM_PATH_3="$BASE_DIR/$PROGRAM_NAME_3"
```
Suggested change:
```diff
+ PROGRAM_NAMES=("read_ultralight" "button_listener" "screen.py")
+ for idx in ${!PROGRAM_NAMES[@]}; do
+    PROGRAM_PATHS[$idx]=("${BASE_DIR}/${PROGRAM_NAMES[$idx]}")
```

Reply:
Looks good